### PR TITLE
[kokoro] Add website generator command

### PR DESCRIPTION
**Pull request by jverkoey**
This will allow our PRs to generate the documentation website as part of our presubmits. This will let us catch any errors that might otherwise affect the ability for the website to be deployed.

This PR also updates the site generator's git url to an https one so that the kokoro instance can clone the repo.

### CODE REVIEW

**Comment by jverkoey** [comment]
There is currently no way to sensibly preview the assets generated by the CI because all of the urls are absolute and css doesn't load as expected. @shyndman is there a way to change the url generation strategy to be relative?

**Comment by jverkoey** [comment]
e.g. the generated html looks like so:

screen shot 2018-04-26 at 10 19 58 pm

File: .kokoro [hunk 1]
```diff
@@ -174,6 +174,26 @@ run_cocoapods() {
   bash <(curl -s https://codecov.io/bash)
 }
 
+generate_website() {
+  if [ -n "$KOKORO_BUILD_NUMBER" ]; then
+    # Move into our cloned repo
+    cd github/repo
```
Comment (romoore) [inline .kokoro:180]: Should you `cd ..` after this  to restore the previous `pwd`?  I see two directory changes in this function, but only one that goes back to the parent directory.

File: scripts/build_site.sh [hunk 2]
```diff
@@ -19,7 +19,7 @@ args=$*
 
 # Checking prerequisites for folders
 # Getting the link for github repository
-GITHUB_REMOTE="git@github.com:material-components/material-components-site-generator.git"
+GITHUB_REMOTE="https://github.com/material-components/material-components-site-generator.git"
```
Comment (romoore) [inline scripts/build_site.sh:22]: Why are we changing the URL as part of this change? Can you document the reason in the commit message, please?

**Review by romoore — APPROVED** [review]

File: .kokoro (hunk 1 above)
Comment (jverkoey) [reply .kokoro:180]: This cd is for the CI server, which starts not in the repo's root. We generally assume that our scripts are run from the repo's root.

File: scripts/build_site.sh (hunk 2 above)
Comment (jverkoey) [reply scripts/build_site.sh:22]: Done.

File: .kokoro (hunk 1 above)
Comment (romoore) [reply .kokoro:180]: Sure, but shouldn't we leave things the way we found them?

**Pull request merged**